modules/shortcodes/vimeo.php

```diff
@@ -78,7 +78,22 @@ function vimeo_shortcode( $atts ) {
 		$height = round( ( $width / 640 ) * 360 );
 	}
 	
+	/**
+	 * Filter the Vimeo player width.
+	 *
+	 * @since 3.4-beta1
```

Comment:
I would use stable versions only (so 3.4 in this case), since 3.4-beta1 is not really a public release.